modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/InstanceofNode.java

```diff
@@ -22,50 +22,28 @@
 import org.elasticsearch.painless.ClassWriter;
 import org.elasticsearch.painless.MethodWriter;
 import org.elasticsearch.painless.lookup.PainlessLookupUtility;
+import org.elasticsearch.painless.lookup.def;
 import org.elasticsearch.painless.phase.IRTreeVisitor;
 import org.elasticsearch.painless.symbol.WriteScope;
-import org.objectweb.asm.Type;
 
 public class InstanceofNode extends UnaryNode {
     
     /* ---- begin node data ---- */
-
+    
```

Comment:
Fix this ws issue.

Reply:
Fixed for consistency with all the other nodes based on existing blanks lines and sections delineated by comments.